From 9088587111eb47f2ca8f8e1dce74ab88eb7ddc43 Mon Sep 17 00:00:00 2001
From: Victor Kaplansky <victork@redhat.com>
Date: Sun, 28 Feb 2016 15:37:47 +0100
Subject: [PATCH 1/3] virtio: coding style tweak

RH-Author: Victor Kaplansky <victork@redhat.com>
Message-id: <1456673583-705-2-git-send-email-victork@redhat.com>
Patchwork-id: 69514
O-Subject: [PATCH RHEV 7.2.z 1/2] virtio: coding style tweak
Bugzilla: 1315198
RH-Acked-by: Xiao Wang <jasowang@redhat.com>
RH-Acked-by: Thomas Huth <thuth@redhat.com>
RH-Acked-by: Michael S. Tsirkin <mst@redhat.com>

From: "Michael S. Tsirkin" <mst@redhat.com>

no space needed after *.

Signed-off-by: Michael S. Tsirkin <mst@redhat.com>
Signed-off-by: Victor Kaplansky <victork@redhat.com>
(cherry picked from commit 850d00700ba787988b6c5404e8c1a3add7141db1)
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 include/hw/virtio/virtio.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/hw/virtio/virtio.h b/include/hw/virtio/virtio.h
index 794c4f3..76a1fc8 100644
--- a/include/hw/virtio/virtio.h
+++ b/include/hw/virtio/virtio.h
@@ -90,7 +90,7 @@ struct VirtIODevice
     VMChangeStateEntry *vmstate;
     char *bus_name;
     uint8_t device_endian;
-    QLIST_HEAD(, VirtQueue) * vector_queues;
+    QLIST_HEAD(, VirtQueue) *vector_queues;
 };
 
 typedef struct VirtioDeviceClass {
-- 
1.8.3.1

